src/main/java/org/broadinstitute/dsde/consent/ontology/resources/MatchResource.java

```diff
@@ -29,23 +14,28 @@
 import javax.ws.rs.container.Suspended;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
-import java.net.URL;
-import java.util.Collection;
-import java.util.List;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.broadinstitute.dsde.consent.ontology.actor.MatchWorkerMessage;
+import org.broadinstitute.dsde.consent.ontology.actor.OntModelCache;
+import org.broadinstitute.dsde.consent.ontology.datause.DataUseMatcher;
+import org.broadinstitute.dsde.consent.ontology.resources.model.DataUse;
+import org.broadinstitute.dsde.consent.ontology.resources.model.DataUseMatchPair;
+import org.broadinstitute.dsde.consent.ontology.service.StoreOntologyService;
 
 @Path("/match")
 @Consumes("application/json")
 @Produces("application/json")
 public class MatchResource {
 
-    private final Logger log = Utils.getLogger(this.getClass());
-    private StoreOntologyService storeOntologyService;
-    private DataUseMatcher dataUseMatcher;
-
-    private static final ActorSystem actorSystem = ActorSystem.create("actorSystem");
-    private static final ActorRef matchWorkerActor = actorSystem.actorOf(Props.create(MatchWorkerActor.class), "MatchWorkerActor");
+    private final StoreOntologyService storeOntologyService;
+    private final DataUseMatcher dataUseMatcher;
+    private final OntModelCache ontModelCache = OntModelCache.INSTANCE;
 
-    MatchResource() {}
+    @Inject
+    MatchResource(DataUseMatcher dataUseMatcher, StoreOntologyService storeOntologyService) {
```

Comment:
Moved to constructor injection per https://github.com/google/guice/wiki/MinimizeMutability